src/test/unit/variational/hier_logistic_cp_test.cpp

```diff
@@ -29,15 +37,14 @@ TEST(advi_test, hier_logistic_cp_constraint_meanfield) {
   // ADVI
   stan::variational::advi<Model_cp, stan::variational::normal_meanfield, rng_t> test_advi(my_model,
                                                      cont_params,
-                                                     10,
+                                                     1,
                                                      100,
-                                                     0.01,
                                                      base_rng,
                                                      100,
                                                      1,
-                                                     &std::cout,
-                                                     &std::cout,
-                                                     &std::cout);
+                                                     &print_stream_,
+                                                     &output_stream_,
+                                                     &diagnostic_stream_);
 
-  test_advi.run(1,2e4);
+  test_advi.run("0.1", 0.1, 1e4, 50);
```

Comment:
same here... make sure it actually does something.

This really allows someone to come in and mess up the `run` function and none of these tests will tell the person that something bad has happened.